release.go

```diff
@@ -248,11 +249,34 @@ func buildReleaseCommits(ctx context.Context, ghClient *githubClient, commits []
 			return nil, err
 		}
 		for i := range v {
+			sha := *v[i].MergeCommitSHA
+			shaPRs[sha] = v[i]
 			number := *v[i].Number
-			prs[number] = v[i]
+			numPRs[number] = v[i]
 		}
 	}
 
+	getPullRequest := func(commit Commit) (*github.PullRequest, error) {
+		if !commit.IsMerge() {
+			return nil, nil
+		}
+		if pr, ok := shaPRs[commit.Hash]; ok {
+			return pr, nil
+		}
+		prNumber, ok := commit.PullRequestNumber()
+		if !ok {
+			return nil, nil
+		}
+		if pr, ok := numPRs[prNumber]; ok {
+			return pr, nil
+		}
```

Comment:
Can we remove this logic by above improvement?

Reply:
Yes! deleted on https://github.com/pipe-cd/actions-gh-release/pull/26/commits/8dde46a44c7a73ee4bdb3836f265df359c3ed5d0.